microsoft-365/commerce/subscriptions/change-plans-manually.md

```diff
@@ -21,7 +21,7 @@ description: "Change subscriptions manually by buying a new subscription and ens
 ROBOTS: NOINDEX
 ms.date: 03/17/2021
 ---
-# Change plans manually
+# Change Microsoft plans manually
```

Comment:
@lhcole613 - I would move the adverb to before the verb that it modifies: "Manually change Microsoft plans"

Reply:
Done